alacritty/src/event.rs

```diff
@@ -552,6 +552,14 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
     fn scheduler_mut(&mut self) -> &mut Scheduler {
         self.scheduler
     }
+
+    fn to_string(&self) -> String {
+        self.terminal.grid_to_string()
+    }
```

Comment:
The `to_string` method should never be manually implemented like this. The `Display` trait should be used instead.

Reply:
That is correct but I don't think we should do it anymore because the signature of `to_string()` has changed.

Reply:
Honestly, I think it is better to just remove the functions from the action context and use `grid_to_string()` directly.